src/main/java/org/jabref/gui/entryeditor/citationrelationtab/RelatedEntriesCache.java

```diff
@@ -0,0 +1,25 @@
+package org.jabref.gui.entryeditor.citationrelationtab;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.identifier.DOI;
+
+public class RelatedEntriesCache {
+    private final Map<String, List<BibEntry>> RELATED_ENTRIES_MAP = new HashMap<>();
+
+    public List<BibEntry> lookupRelatedEntries(BibEntry entry) {
+        return RELATED_ENTRIES_MAP.getOrDefault(entry.getDOI().map(DOI::getDOI).orElse(""), Collections.emptyList());
```

Comment:
The implementation is "OK", because it fits in one line. Not sure, what the other thigns about the "misuse" of `"""`.

Reply:
An empty string is an invalid DOI for which `DOI.getDOI()` is expected to return `Optional.empty()`.So it should be impossible to find an element in the map with an empty string as key. I'm not sure what you mean by misuse.

Reply:
I ended up with this:
```java
entry.getDOI().map(DOI::getDOI)
                .map(RELATED_ENTRIES_MAP::get)
                .orElse(Collections.emptyList());
```